integrations/tensorflow/test/requirements.txt

```diff
@@ -1,15 +1,4 @@
 # Requirements for running TF tests
-tensorflow>=2.7.1
+tf-nightly
```

Comment:
Should we pin to a specific one?

Reply:
I guess we could temporarily.  All we're doing really is waiting for the next TF release, at which point we should be able to get back on the main TF releases.

Reply:
Yes, my concern is just that the testing of nightly is lower, so I'd prefer not flakes from TF updates and I'm ok to pinning to any working one until next release.

Reply:
Makes sense, updated